dart/test/sentry_test.dart

```diff
@@ -489,7 +489,7 @@ void main() {
     );
 
     // ignore: deprecated_member_use_from_same_package
```

Comment:
We can also remove the `// ignore: deprecated_member_use_from_same_package` directives.